src/kirby/components/checkbox/checkbox.component.ts

```diff
@@ -0,0 +1,43 @@
+import { Component, OnInit, Input, Output, EventEmitter } from '@angular/core';
+
+declare var require: any;
+const style: any = require('sass-extract-loader!./checkbox.component.scss');
+
+@Component({
+  selector: 'kirby-checkbox',
+  templateUrl: './checkbox.component.html',
+  styleUrls: ['./checkbox.component.scss'],
+})
+export class CheckboxComponent implements OnInit {
```

Comment:
Consider adding `changeDetection: ChangeDetectionStrategy.OnPush` (to decorator), since state is pushed to Component through `@Input()`-decorated properties.